.github/actions/check-image-and-changes/action.yaml

```diff
@@ -43,15 +46,38 @@ runs:
         BASE_COMMIT=${{ github.event.pull_request.base.sha }}
         HEAD_COMMIT=${{ github.event.pull_request.head.sha }}
 
+        # Check if commit message contains [skip-build]
+        COMMIT_MSG=$(git log -1 --pretty=%B)
+        if [[ "$COMMIT_MSG" == *"[skip-build]"* ]]; then
+          echo "Commit message contains [skip-build], marking as skipped"
+          echo "is_skipped=true" >> $GITHUB_OUTPUT
+          echo "relevant=false" >> $GITHUB_OUTPUT
+          exit 0
+        fi
+
+        # Default to not skipped
```

Comment:
this should be done before the if block, and since you ALSO want to check for relevant changes and an existing image, you don't want to exit too early (above).